.storybook/webpack.config.ts

```diff
@@ -24,6 +24,9 @@ const basePathMap = generatePathMap(genPath)
 // directories, so that we can replace things in Storybook.
 const pathMap = {
   ...basePathMap,
+  '//resources/mojo/mojo/public': [
```

Comment:
Interesting - do we not need to map `chrome://resources/mojo/mojo/public` too?

Reply:
Fixed.